web-ui/src/main/resources/catalog/geocat-shared-objects/js/extent-directive.js

```diff
@@ -89,8 +89,14 @@
                * @geom {ol.geometry}
                */
               scope.fillInput = function() {
-                var geom = featureOverlay.getFeatures().item(0).getGeometry().
-                    clone().transform(map.getView().getProjection(), scope.formObj.proj);
+                var feature = featureOverlay.getFeatures().item(0) || scope.feature;
+                if (!feature) {
+                  return;
+                }
+                var geom = feature.getGeometry().clone();
+                if (!scope.incompatible) {
```

Comment:
Do you set it back to `false` somehow, for example if afterwards the user draws a bbox ?